src/services/build-service.ts

```diff
@@ -0,0 +1,50 @@
+import * as path from 'node:path'
+import * as fs from 'node:fs'
+import { CLIError } from '@oclif/errors'
+import debugFactory from './debug-factory-service'
+
+import { MICROSERVICES_FOLDER } from '../paths'
+import ProcessExecutorService from './process-executor-service'
+import { ComponentService } from './component-service'
+
+export class BuildService {
```

Comment:
Do we need a separate `BuildService` class just for the build method/s?

@cecchisandrone What do you think about the `build()` method being in `ComponentService` instead?

Reply:
Build moved to ComponentService